app/components/task-list/multiple-sections.njk

```diff
@@ -0,0 +1,118 @@
+{% set html_style = 'background-color: #f0f4f5;' %}
+{% set title = 'Task list' %}
+{% from 'components/task-list/macro.njk' import taskList %}
+{% from 'components/button/macro.njk' import button %}
+{% extends 'layout.njk' %}
+
+{% block body %}
+
+  <div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper" id="maincontent">
+
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+          <h1 class="nhsuk-heading-l">Service name</h1>
+
+          <h2 class="nhsuk-heading-m">Check before you start</h2>
+          {{ taskList({
+            idPrefix: "before-you-start",
+            items: [
+              {
+                title: {
+                  text: "Check eligibility"
+                },
+                href: "#",
+                status: {
+                  text: "Completed",
+                  classes: "nhsuk-task-list__status--completed"
+                }
+              },
+              {
+                title: {
+                  text: "Read declaration"
+                },
+                href: "#",
+                status: {
+                  tag: {
+                    text: "Incomplete",
+                    classes: "nhsuk-tag--blue"
+                  }
+                }
+              }
+            ]
+          }) }}
+
+          <h2 class="nhsuk-heading-m nhsuk-u-margin-top-3">Prepare application</h2>
```

Comment:
I think this override class can be removed, as it’s not making a difference? The margin-top is adding to the heading is smaller than the margin-bottom on the task list above, and the margins are collapsed.

```suggestion
          <h2 class="nhsuk-heading-m">Prepare application</h2>
```